seccomp/src/lib.rs

```diff
@@ -942,27 +922,11 @@ impl SeccompFilterContext {
 ///
 /// * `level` - Filtering level.
 ///
-pub fn setup_seccomp(level: SeccompLevel) -> Result<()> {
+pub fn setup_seccomp(context: SeccompFilterContext) -> Result<()> {
```

Comment:
Why not have this as part of the filter struct? I.e. `SeccompFilterContext::apply()`.